### ISSUE TITLE
New Dhcp Scopes check. 

### ISSUE THREAD

**Issue by drapiti** [Icinga/icinga-powershell-plugins#254]
Would it be possible to add an official dhcp check to verify the occupation of dhcp leases on specific windows dhcp servers. It would be great if we could have something simple like this:

$scope = Get-DhcpServerV4Scope
$statistics = $scope | % {Get-DhcpServerv4ScopeStatistics -ScopeId $_.ScopeID}
[Array]$allert = @() 
$allert = $statistics |? {$_.PercentageInUse -ge 98} 

if ($allert.count -ge 1) {
    $message = $allert | out-String
    Write-EventLog -LogName Application -EntryType Error -EventId 4444 -Source monitoringDhcp -Message $message
}
else {
    Write-EventLog -LogName Application -EventId 5555 -Source monitoringDhcp -Message "No scopes are full..."
}

An event log is not necessary obviously but something that simply checks each scope is required. 

Thanks.

**Comment by LordHepipud** [comment]
Yeah this is something we can definitely provide. I will have a look on this and put it in the roadmap for v1.9.0
